syft/pkg/cataloger/binary/default_classifiers.go

```diff
@@ -196,4 +196,24 @@ var defaultClassifiers = []classifier{
 		Package: "postgresql",
 		PURL:    mustPURL("pkg:generic/postgresql@version"),
 	},
+	{
+		Class:    "rust-standard-library-linux",
+		FileGlob: "**/libstd-????????????????.so",
```

Comment:
good call, that is more specific and not too crazy for the index parsing in stereoscope 🙌